desktop/pinentry-bemenu/pinentry-bemenu.info

```diff
@@ -0,0 +1,10 @@
+PRGNAM="pinentry-bemenu"
+VERSION="0.14.0"
+HOMEPAGE="https://github.com/t-8ch/pinentry-bemenu"
+DOWNLOAD="https://github.com/t-8ch/pinentry-bemenu/archive/refs/tags/v0.14.0.tar.gz"
```

Comment:
Try: https://github.com/t-8ch/pinentry-bemenu/archive/v0.14.0/pinentry-bemenu-0.14.0.tar.gz

https://slackbuilds.org/GITHUB_URLs.txt

Reply:
Thx - adjusted and resubmitted

https://slackbuilds.org/pending/